ee/clickhouse/queries/funnels/test/__snapshots__/test_funnel_correlation.ambr

```diff
@@ -0,0 +1,442 @@
+               argMax(latest_0, steps) as first_timestamp
+        FROM
+          (SELECT aggregation_target,
+                  steps,
+                  max(steps) over (PARTITION BY aggregation_target) as max_steps,
+                                  step_1_conversion_time ,
+                                  latest_0,
+                                  latest_1,
+                                  latest_0
+           FROM
+             (SELECT *,
+                     if(latest_0 < latest_1
+                        AND latest_1 <= latest_0 + INTERVAL 14 DAY, 2, 1) AS steps ,
+                     if(isNotNull(latest_1)
+                        AND latest_1 <= latest_0 + INTERVAL 14 DAY, dateDiff('second', toDateTime(latest_0), toDateTime(latest_1)), NULL) step_1_conversion_time
+              FROM
+                (SELECT aggregation_target,
+                        timestamp,
+                        step_0,
+                        latest_0,
+                        step_1,
+                        min(latest_1) over (PARTITION by aggregation_target
+                                            ORDER BY timestamp DESC ROWS BETWEEN UNBOUNDED PRECEDING AND 0 PRECEDING) latest_1
+                 FROM
+                   (SELECT aggregation_target,
+                           timestamp,
+                           if(event = 'user signed up', 1, 0) as step_0,
+                           if(step_0 = 1, timestamp, null) as latest_0,
+                           if(event = 'paid', 1, 0) as step_1,
+                           if(step_1 = 1, timestamp, null) as latest_1
+                    FROM
+                      (SELECT e.event as event,
+                              e.team_id as team_id,
+                              e.distinct_id as distinct_id,
+                              e.timestamp as timestamp,
+                              e.$group_0 as aggregation_target,
+                              groups_0.group_properties_0 as group_properties_0
+                       FROM events e
+                       INNER JOIN
+                         (SELECT distinct_id,
+                                 argMax(person_id, _timestamp) as person_id
+                          FROM
+                            (SELECT distinct_id,
+                                    person_id,
+                                    max(_timestamp) as _timestamp
+                             FROM person_distinct_id
+                             WHERE team_id = 2
+                             GROUP BY person_id,
+                                      distinct_id,
+                                      team_id
+                             HAVING max(is_deleted) = 0)
+                          GROUP BY distinct_id) AS pdi ON events.distinct_id = pdi.distinct_id
+                       INNER JOIN
+                         (SELECT group_key,
+                                 argMax(group_properties, _timestamp) AS group_properties_0
```

Comment:
Do a random test somewhere with a different group type index.

Reply:
This is annoying? Group_properties were queried in the funnel, but they're only needed in the correlation query

Reply:
that is, remove the funnel props from the CTE generator for funnels